examples/tutorial.py

```diff
@@ -24,27 +32,26 @@
 # - key features of daisy that make it unique
 
 # %%
-# %pip install scikit-image
-# %pip install zarr
-# %pip install matplotlib
-# %pip install funlib.persistence
+# Don't forget to install daisy with dependencies for the docs before running this tutorial:
+#
+# `pip install daisy[docs]``
 
 # %% [markdown]
-# # Building Blocks
+# ## Building Blocks
 
 # %% [markdown]
 # Daisy is designed for processing volumetric data. Therefore, it has specific ways to describe locations in a volume. We will demonstrate the common terms and utilities using this image of astronaut Eileen Collins.
 
 # %%
-from skimage import data
-import numpy as np
-import matplotlib.pyplot as plt
+from skimage import data  # noqa
+import numpy as np  # noqa
+import matplotlib.pyplot as plt  # noqa
 
 raw_data = np.flip(data.astronaut(), 0)
 axes_image = plt.imshow(raw_data, zorder=1, origin="lower")
 
 # %% [markdown]
-# ## Coordinate
+# ###Coordinate
```

Comment:
This no longer shows up as a header in the docs - I think the space was important. This comment applies to all the updated headings below as well

Reply:
Done